### PR TITLE
switch aplcms to conda

### PR DESCRIPTION
**Pull request by martenson**

### CODE REVIEW

**Comment by maximskorik** [comment]
@martenson I don't think anyone has added functionality to export output data to hdf format to this wrapper. The exporter has been removed from the https://github.com/RECETOX/recetox-aplcms along with HDF5 dependency so the tool isn't going to work in Biocontainer yet. I'll make an issue.

File: tools/aplcms/aplcms_macros.xml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <macros>
-    <token name="@TOOL_VERSION@">6.6.6</token>
+    <token name="@TOOL_VERSION@">6.6.8-recetox0</token>
 
     <xml name="requirements">
         <requirements>
-            <container type="docker">recetox/aplcms:6.6.6-recetox2</container>
+            <requirement type="package" version="0">r-recetox-aplcms</requirement>
```
Comment (maximskorik) [inline tools/aplcms/aplcms_macros.xml:6]:
```suggestion
            <requirement type="package" version="0">r-recetox-aplcms</requirement>
            <requirement type="package" version="1.3.2">r-hdf5r</requirement>

```

Comment (maximskorik) [inline tools/aplcms/aplcms_macros.xml:6]:
```suggestion
            <requirement type="package" version="1.3.2">r-hdf5r</requirement>

```

Comment (maximskorik) [inline tools/aplcms/aplcms_macros.xml:6]:
```suggestion
            <requirement type="package" version="0">r-recetox-aplcms</requirement>
            <requirement type="package" version="1.3.2">r-hdf5r</requirement>

```

Comment (maximskorik) [reply tools/aplcms/aplcms_macros.xml:6]: Also add HDF5 dependency since it's not present in r-recetox-aplcms package.

**Comment by maximskorik** [comment]
https://github.com/RECETOX/galaxytools/issues/111

File: tools/aplcms/aplcms_macros.xml (hunk 1 above)
Comment (maximskorik) [reply tools/aplcms/aplcms_macros.xml:6]: Okay, this may be unnecessary due to https://github.com/RECETOX/recetox-aplcms/issues/13

**Comment by martenson** [comment]
super seded by https://github.com/RECETOX/galaxytools/pull/133

**Pull request closed without merging**